qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -202,6 +202,8 @@ def install_deps(self):
             if self.install_xfsprogs:
                 if distro == 'centosstream' and major_ver_num == 8:
                     deps += ['--enablerepo=powertools']
+                if distro == 'centosstream' and major_ver_num == 9:
```

Comment:
Can't we just switch to `python3` in the deps list above? The ubuntu distro case seems to be correctly handled below.

Reply:
+1, anyways py2 is deprecated. py3 is what we should default to

Reply:
Okay, sounds reasonable.